docker/templates/django/{{cookiecutter.app_name}}/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "{{cookiecutter.app_name}}",
+  "version": "0.0.0",
+  "description": "JavaScript development setup for {{cookiecutter.app_name}}",
+  "dependencies": {
+    "babel-preset-env": "^1.7.0",
+    "babelify": "^7.3.0",
+    "browserify": "^16.5.0"
```

Comment:
These dependencies are kind of old, but I tried updating them (particularly updating `babel-preset-env` to the new `@babel/preset-env`) and found they didn't work. What's the maintenance outlook on these dependencies? And is there active development on https://github.com/kottenator/django-compressor-toolkit?